src/pages/perfilPaciente/PerfilPacientePage.tsx

```diff
@@ -44,7 +45,7 @@ const PerfilPaciente = () => {
         <Nav>
           <p>Perfil</p>
           <p>Fases</p>
-          <p>Vacinas</p>
+          <p onClick={() => navigate(`/vacinas/paciente/5f59f128-148d-469b-9452-4613e6afd67c`)}>Vacinas</p>
```

Comment:
Esse código é fixo mesmo? Casos seja criar uma constante com o valor.

Reply:
Não, mas como não tem integração com o BE eu deixei uma string que funciona. Pretendo mudar na próxima atualização.